pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -323,7 +328,7 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 			// CPUs on a physical core. Allocation of individual threads would never have to occur.
 			return SMTAlignmentError{
 				RequestedCPUs: numCPUs,
-				CpusPerCore:   CPUsPerCore,
+				CpusPerCore:   p.cpuGroupSize,
```

Comment:
nit:
```suggestion
				CpusPerCore:   p.cpuGroupSize,
				CausedByPhysicalCPUs:  false,
```
@ffromani up to you to consider or not. I found it a little bit more expressive and explicit.

Reply:
added. Not sure it helps, but not against either so why not?